test/unit/pointcloudlayerparsing.js

```diff
@@ -1,51 +1,93 @@
 import assert from 'assert';
-import PointCloudLayer, { parseMetadata } from 'Layer/PointCloudLayer';
+import PointCloudLayer from 'Layer/PointCloudLayer';
+import PointCloudSource from 'Source/PointCloudSource';
+import Coordinates from 'Core/Geographic/Coordinates';
+import GlobeView from 'Core/Prefab/GlobeView';
+import HttpsProxyAgent from 'https-proxy-agent';
+import Renderer from './mock';
 
-describe('PointCloudProvider', function () {
-    it('should correctly parse normal information in metadata', function () {
-        const layer = {
-            material: { defines: {} },
-        };
+const renderer = new Renderer();
```

Comment:
Please move those variables (`renderer`,  `placement`...) between the `describe` and `it` section, so it stays in this scope.

Reply:
ok